src/Views/Basket/Basket.js

```diff
@@ -1,11 +1,118 @@
-import React from 'react'
+import { useState, useEffect } from "react";
+import { useLocation } from "react-router-dom";
 
-const Basket  = () => {
-    return ( 
+import { searchMyBooks } from "../../API/fetchBooks";
+import Book from "../../SharedComponents/Book";
+import BasketCreator from "./BasketCreator";
+import addBooksToList from "../../SharedFunctions/addBooksToList";
+import getPagination from "../../SharedFunctions/getPagination";
+import Pagination from "../../SharedComponents/Pagination";
+
+import "./Basket.scss";
+
+const CreateBasket = ({ accessToken }) => {
+  const location = useLocation();
+  const user = location.state.user;
+  const chosenBooks = location.state.chosenBooks;
+  const [offeredBooks, setOfferedBooks] = useState([]);
+  const [myBooks, setMyBooks] = useState([]);
+  const [basketCreatorVisible, setBasketCreatorVisible] = useState(false);
+  const [chosenPage, setChosenPage] = useState(1);
+  const [offeredPage, setOfferedPage] = useState(1);
+  const onPageLimit = 10;
+
+  useEffect(() => {
+    const getBooks = async () => {
+      const res = await searchMyBooks({
+        Authorization: `Bearer ${accessToken}`,
+      });
```

Comment:
why pass whole object with Authorization header, when you can pass only accessToken and the logic associated with header have only inside of `searchMyBooks` method?